### DISCUSSION TITLE
I was blocked by Telegram Bot.

### DISCUSSION THREAD

**Discussion by mufasa71** [feature-sliced/documentation#530] [Q&A] [answered]
There was time to react on telegram bot which i missed. How to be unblocked?

**Comment by azinit** [comment]
@unordinarity

**Comment by unordinarity** [comment, marked as answer]
@mufasa71 message me in telegram `@unordinarity`

And sorry for those anti-spam measures, sometimes these are too strong

**Reply by mufasa71** [reply]
Solved, thanks!
